infrastructure/modules/armonik/ingress-configmap.tf

```diff
@@ -38,13 +44,20 @@ server {
 
     sendfile on;
 
+    if ($accept_language ~ "^$") {
+	set $accept_language "en";
+    }
+
     location = / {
-        rewrite ^ $scheme://$http_host/admin/ permanent;
+        rewrite ^ $scheme://$http_host/admin/$accept_language/ permanent;
     }
     location = /admin {
-        rewrite ^ $scheme://$http_host/admin/ permanent;
+        rewrite ^ $scheme://$http_host/admin/$accept_language/ permanent;
+    }
+    location = /admin/ {
+	rewrite ^ $scheme://$http_host/admin/$accept_language/ permanent;
     }
-    location /admin/ {
+    location ~ /admin/(fr|en) {
```

Comment:
Do you really need the regexp?

Reply:
The used of the regex is to make sure we only redirect to the availables languages of the GUI.

Reply:
This can be catched by the backend. We will never redirect something that starts with `/admin/`but goes somewhere else.